package.json

```
@@ -5,7 +5,9 @@
   "license": "MIT",
   "scripts": {
     "start": "npm run build && node dist/index.js",
-    "build": "tsc"
+    "build": "tsc",
+    "lint": "tslint --project ./",
+    "fix": "yarn lint -- -fix"
```

What do you think about? 
```suggestion
    "lint:fix": "yarn lint -- -fix"
```